src/components/editMeasure/measureDetails/measureInformation/MeasureInformation.tsx

```diff
@@ -46,6 +46,10 @@ export default function MeasureInformation() {
         <span tw="mr-2">Measure Name:</span>
         <InlineEdit text={measure.measureName} onSetText={updateMeasureTitle} />
       </div>
+      <div tw="flex">
+        <span tw="mr-2">Measure CQL Library Name:</span>
+        <span>{measure.cqlLibraryName || "NA"}</span>
+      </div>
```

Comment:
If you can add a data-testId, that will help QA with regression.

Reply:
let me add that, thanks!